src/com/company/Main.java

```diff
@@ -8,18 +8,40 @@
 
 
 public class Main {
-    public static void main(String[] args){
+    public static void main(String[] args) {
+
+        String txtFilePath = args[0];
+
+        File file = new File(txtFilePath);
+        if(file.length()==0)
```

Comment:
```suggestion
        if(file.length()==0) {
```
curly bracket should be in the same line

Reply:
done that for all the lines which has this scenarios.